test/e2e/conformance/cross_logical_cluster_list_test.go

```diff
@@ -134,26 +129,82 @@ func TestCrossLogicalClusterListPartialObjectMetadata(t *testing.T) {
 
 	cfg := server.DefaultConfig(t)
 
+	// Make sure the informers aren't throttled
+	cfg.QPS = 500
+	cfg.Burst = 1000
+
```

Comment:
why not?

Reply:
This is just for testing because it is starting the dynamic shared informer factory and it makes lots of discovery calls. Shortens test execution time.

Reply:
ack

Reply:
add a sentence in the comment "... because dynamic informers does lots of discovery which slows down tests"